src/tests/test_utils/unit_test_utils/mocks/cpp_interfaces/impl/mock_inference_plugin_internal.hpp

```diff
@@ -4,26 +4,30 @@
 
 #pragma once
 
+#include <gmock/gmock.h>
+
+#include <cpp_interfaces/interface/ie_iexecutable_network_internal.hpp>
+#include <cpp_interfaces/interface/ie_iplugin_internal.hpp>
```

Comment:
""

Reply:
These files will be removed in 2024.0 release, it is why I kept at as is :)